payavel/serviceable
# Allow for stub customization.

### **What does this PR do?** :robot:
- Adapts existing commands to support customization of stub files.
- Adds `service:stubs` command to publish stub files.

### **How should this be tested?** :microscope:
1. Run `php artisan service:stubs`.
2. Modify some of the generated stub files.
3. Run `php artisan service:install`.
4. Verify that the command is utilizing your custom stub files.

### **Any background context you would like to provide?** :construction:
Attempted to make it as close to the "[Laravel Way](https://laravel.com/docs/master/artisan#stub-customization)" as possible. 

### **Does this relate to any issue?** :link:
Closes #2
